lib/lib.es5.d.ts

```diff
@@ -964,7 +973,11 @@ interface RegExpConstructor {
     $7: string;
     $8: string;
     $9: string;
+    input: string;
```

Comment:
These 4 properties are marked as non-standard. I don't want to make our list of non-standard extensions any bigger than it already is.